src/Components/AddClassOnline/AddClassOnline.tsx

```diff
@@ -0,0 +1,35 @@
+import styles from "./AddClassOnline.module.css";
+import Clock from "./assets/clock.svg";
+import Location from "./assets/location.svg";
+import Plus from "./assets/plus.svg";
+
+const classes = [
+  { name: "abs & booty camp", time: " 13:00", place: " online" },
+  { name: "weight circuit", time: " 18:00", place: " online" },
+];
+
+export default function AddClassOnline() {
+  return (
+    <>
+      {classes.map(({ name, time, place }) => (
+        <article className={styles.singleCard} key={name}>
+          <h4 className={styles.courseName}>{name}</h4>
+          <div className={styles.withButton}>
+            <div className={styles.allInfo}>
+              <span className={styles.info}>
+                <img src={Clock} alt="Clock Icon" />
```

Comment:
In this case, for me the alt text "clock icon" is not the best way to describe the text that follows. Maybe "start time"?